### ISSUE TITLE
What are KMS requests on AWS and why they keep increasing?

### ISSUE THREAD

**Issue by mohitkyadav** [iiitv/aua#13]
I was charged INR 7.83 in June for exceeding free tier quota limit for KMS requests(20,000 / month).
In July I've used 45 % of my KMS monthly quota. I terminated all instances deleted all security groups(except default ones) yesterday but still, KMS request count increased from 34 % to 45 %.
Can someone tell me what are KMS requests and why they keep increasing?

**Comment by mohitkyadav** [comment]
Resolved.

**Comment by aviaryan** [comment]
Please post the answer @mohitkyadav. This may help others.

**Comment by mohitkyadav** [comment]
The problem was caused by a S3 bucket, running in region Ohio. When I was trying to deploy Odyssy, I was figuring out every option, there are a lot of options, links and services at first glance those seem too confusing. I deployed it 3 times using CodeStar, CodeCommit and a custom Ubuntu server instance. After realizing the unusual growth in KMS requests I deleted the CodeStar and CodeCommit deployments but I forgot about the storage. So at last I deleted the S3 bucket and deactivated all regions except Mumbai.
